bin/ci/install.sh

```diff
@@ -11,6 +11,12 @@
 
 cd "$(dirname "$0")" || exit; source _includes.sh
 
+# Switch to PHP8.
+if [[ "$PHP8_JOB" == TRUE || "$JENKINS_HOME" ]]; then
+  update-alternatives --install /usr/local/bin/php php /usr/bin/php8.0 80
```

Comment:
I don't this is where this belongs since it's making significant platform-level customizations--not just changing paths, like our GitHub Actions and Travis CI support, which are basically identical otherwise. We can do one of two things instead: 1) move the customizations into a platform-specific script like we did for Pipelines, e.g., `bin/jenkins/before_install.sh`, or 2) let the CI platform put the logic in its own configuration or templates. If you'd like, we can discuss the relative merits of the choices at scrum.